jetstream/maxtext/single-host-inference/deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: maxengine-server
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: maxengine-server
+  template:
+    metadata:
+      labels:
+        app: maxengine-server
+    spec:
+      nodeSelector:
+        cloud.google.com/gke-tpu-topology: 2x4
+        cloud.google.com/gke-tpu-accelerator: tpu-v5-lite-podslice
+      containers:
+      - name: maxengine-server
+        image: us-docker.pkg.dev/cloud-tpu-images/inference/maxengine-server:v1.0.0
+        securityContext:
+          privileged: true
+        args:
+        - model_name=gemma-7b
+        - tokenizer_path=assets/tokenizer.gemma
+        - per_device_batch_size=1
+        - run_name=maxtext_jetstream_run
```

Comment:
sorry to comment on this again, but this is still not clear to me and is sticking like a sore thumb :) does the value have meaning? is it associated with the checkpoint or the server binary? how do users discover it? or is it some arbitrary string?

Reply:
Thanks for actively following up with this. Previously I was under the impression that it was needed (because in training it was used) and via some discussions internally. For inferencing, I just tested with omitting run_name and no errors arise so I will omit the run_name from the yaml